test/tedious/tedious.js

```diff
@@ -68,6 +68,7 @@ describe('tedious', () => {
     it('query with multiple errors', done => TESTS['query with multiple errors'](done))
     it('query with raiseerror', done => TESTS['query with raiseerror'](done))
     it('query with pipe', done => TESTS['query with pipe'](done))
+    it('query with pipe and back pressure', (done) => TESTS['query with pipe and back pressure'](done))
```

Comment:
we need to add this to the msnodesqlv8 test suite too.

Reply:
sure, we can do that. I was wondering about that, we can add that.